template/task-templates.wdl

```diff
@@ -28,34 +24,32 @@ task static_disk_and_ram_task {
     }
 
     runtime {
-        memory: "~{memory_gb} GB"
-        disk: "~{disk_size_gb} GB"
+        memory: "<> GB"
+        disk: "<> GB"
         container: ""
```

Comment:
Should we put a reasonable default for `container`? Likely it will almost always be overridden.

Reply:
I think it depends. Pretty much all of our `util` tasks could run in a default container, such as `ubuntu:22.04`. So it isn't wrong in all cases. Are our templates designed to be just a fillable form or do we want to provide a as close to working as possible starting point?

Reply:
Here's my hold-up on providing a default container. We want the container selected for a task to be _targeted_. We have had internal discussions about how much we dislike the "kitchen sink" convention encouraged by certain set ups. A container should be as minimal as possible, right? That seems a good convention to stick by. And by that philosophy, a "default container" isn't very sensible, is it?

But we do have the `util` image, which is admittedly a bit of a kitchen sink container. We aren't always following our own best practices here. (I _DO NOT_ want to split `util` into more targeted containers. That is way more work than it's worth, but I think it's worth acknowledging that touch of hypocrisy we have going on).

Hypothetical scenario: You are an external collaborator looking to add some tasks to this repo as a good samaritan. You have not been privy to all our internal discussions about containers and our philosophy. You start developing with our templates, which have `util` already filled in.
You realize that `util` doesn't cut it for your use case. What's your first instinct here? To shop around for a different extant container that meets your needs? Or modify util? It's the default, right? Shouldn't it meet your needs?
Contrast to the hypothetical where no default container is supplied. First thing you do as a developer would be find a container to suit your use case.

IMO the latter is the correct development pattern. Pick a container first, write your task around what is available in your image. Write a new container (or modify an extant one) only as a last resort.

So I still think "no default container". But I am close to being persuaded here. Do you have any more thoughts?

Reply:
You make good points. I wouldn't use the `util` image as the default, but something public like `ubuntu` or `alpine` (open to others). I'd probably suggest an in-line comment in the template to reinforce the idea that that value should likely be changed by the dev. That said, I can see leaving it empty as well. 

Do you happen to know if an empty string in the `container` field is valid WDL? Or if it is included, it must be a valid container path?